sdk/nodejs/output.ts

```diff
@@ -216,6 +219,9 @@ To get the value of an Output as a JSON value or JSON string consider either:
 
 See https://www.pulumi.com/docs/concepts/inputs-outputs for more details.
 This function may throw in a future version of @pulumi/pulumi.`;
+            if (settings.throwOutputToString()) {
+                throw new Error(message);
```

Comment:
Probably shouldn't include "is function may throw in a future version of @pulumi/pulumi." in the message thrown here.